### ISSUE TITLE
Increase render distsnce?

### ISSUE THREAD

**Issue by foote-darrell** [jorio/Bugdom2#8]
The render distance seems to be lower than I remember. Particularly on _Level 8: The Garbage Can_ and _Level 10: The Park._
![image](https://github.com/jorio/Bugdom2/assets/70550621/4dbd5819-f3b1-4b6f-a60f-7df5d5d55982)
![Screenshot from 2023-11-02 19-20-07](https://github.com/jorio/Bugdom2/assets/70550621/e2844023-a185-4949-b6b3-d26c8f8df26d)
Now there were tutorials for how to increase the render distance in [_Bugdom_](https://github.com/jorio/Bugdom/issues/24), [_Otto Matic,_](https://github.com/jorio/Bugdom/issues/24#issuecomment-1142407535) and [_Nanosaur._](https://github.com/jorio/Nanosaur/issues/22) These didn't seem to work in Bugdom 2 however. [This bloke](https://www.reddit.com/r/PangeaSoftware/comments/174iopk/bugdom_2_with_increased_render_distance_and_other/) was able to do it.

**Comment by orazioedoardo** [comment]
Just took a look at this and works similar to Otto Matic but some levels use ad-hoc supertile active range, yon and fog, so you need to override those as well.

Change [MAX_SUPERTILE_ACTIVE_RANGE](https://github.com/jorio/Bugdom2/blob/69115862fe977859997b417d9044b9eb6388a0f1/Source/Headers/terrain.h#L42) to 50, add `mask = 2;` [here](https://github.com/jorio/Bugdom2/blob/69115862fe977859997b417d9044b9eb6388a0f1/Source/Terrain/Terrain.c#L1681) to override previous mask in switch case and add these lines [here](https://github.com/jorio/Bugdom2/blob/69115862fe977859997b417d9044b9eb6388a0f1/Source/System/Main.c#L688), again to override values in previous switch case.

```
gSuperTileActiveRange           = MAX_SUPERTILE_ACTIVE_RANGE;
viewDef.styles.useFog           = true;
viewDef.view.clearColor.r       = 1;
viewDef.view.clearColor.g       = 1;
viewDef.view.clearColor.b       = 1;
viewDef.view.clearBackBuffer    = true;
viewDef.camera.fov              = GAME_FOV;
viewDef.camera.yon              = (gSuperTileActiveRange * SUPERTILE_SIZE * gTerrainPolygonSize) * .95f;
viewDef.styles.fogStart         = viewDef.camera.yon * .7f;
viewDef.styles.fogEnd           = viewDef.camera.yon * 1.0f;
gDrawLensFlare = true;
```
![level_10](https://github.com/jorio/Bugdom2/assets/22700499/e500dca0-b0fd-48a6-a2d4-c0f44ebca8a2)

By the way, I had never noticed that level 10 terrain was not flat.

**Comment by foote-darrell** [comment]
Wow, that looks much better.
![Screenshot from 2023-11-12 14-25-45](https://github.com/jorio/Bugdom2/assets/70550621/c96bb194-b70d-47bb-b066-ad3daf9c2022)
![Screenshot from 2023-11-12 14-59-46](https://github.com/jorio/Bugdom2/assets/70550621/0b73b359-04c5-4911-a3c5-b8b14857ff9e)
The water slide levels still seem to be short-sighted. Is this controlled differently?
![Screenshot from 2023-11-12 14-25-03](https://github.com/jorio/Bugdom2/assets/70550621/2fe390d1-f8be-4019-8f35-28bb48112386)
![Screenshot from 2023-11-12 14-24-50](https://github.com/jorio/Bugdom2/assets/70550621/ec20544d-8562-4019-8d71-53780321cb25)

**Comment by orazioedoardo** [comment]
> The water slide levels still seem to be short-sighted. Is this controlled differently?

Yeah, you need to add the same overrides in my code block after [this if statement](https://github.com/jorio/Bugdom2/blob/69115862fe977859997b417d9044b9eb6388a0f1/Source/Player/Player_Tunnel.c#L118-L173).

![gutter](https://github.com/jorio/Bugdom2/assets/22700499/f531aef8-0384-4cad-b2ad-6b51759093cd)

**Comment by EliasHeydrich** [comment]
Will this trigger additional AI actors earlier, as it did with Otto-Matic and Bugdom 1?
Also, is there a pre-compiled build for it?
